### PR TITLE
Added WSL2 in installation guide

### LINKED ISSUE #924: [DOC]Install requirements for WSL2 in installation guide
#### Describe the issue linked to the documentation
It has come to my attention that in order to run `sktime` under WSL2, we should follow the Ubuntu linux instructions.

We should include a `..note` directive in the windows instructions pointing to the linux ones for users of WSL2 as their platform.

### PR DESCRIPTION
**Pull request by afzal442**
#### Reference Issues/PRs
Fixes #924

#### What does this implement/fix? Explain your changes.

Added WSL-2 installation note for any user who wants to experience LINUX terminal feat on windows machine
##### For all contributions
- [x] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).
- [ ] Optionally, I've updated sktime's [CODEOWNERS](https://github.com/alan-turing-institute/sktime/blob/main/CODEOWNERS) to receive notifications about future changes to these files.
- [ ] I've added unit tests and made sure they pass locally.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -224,9 +224,17 @@ And build sktime from this environment:
...
+    Some users have experienced issues when installing NumPy, particuarly version 1.19.4. Note that a recent Windows update may affect compilation using Visual Studio (see `Windows update issue <https://developercommunity.visualstudio.com/content/problem/1207405/fmod-after-an-update-to-windows-2004-is-causing-a.html>`_).
+
+.. note::
+
+    If you want to use linux terminal on a window machine, you can download wsl-2 to experience the same features as linux Ubuntu OS,
+    Make sure you have Ubuntu 20.04 installed on your windows and follow the instructions to install wsl-2 from `here <https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-2---check-requirements-for-running-wsl-2>`_ and run commands as mentioned above on LINUX section. Maybe you need to install fbprophet dependency at some point using
+    ``pip3 install fbprophet``.
 
-If you run into a problem installing the development version and are using Anaconda, try:
+If you run into a problem installing the development version and are using Anaconda, plz, try out these steps as follows:
```
Comment (fkiraly) [inline docs/source/installation.rst:237]: this is odd English, I think the old line was better

Comment (afzal442) [reply docs/source/installation.rst:237]: Thanks, @fkiraly, I will revert that. Could you plz review other content as well?

**Comment by fkiraly** [comment]
Thanks! Had another look at it.

In the addition, I find the instructions a bit vague. "maybe install facebook prophet"? This is not very specific. Can you be a bit more precise please, under which condition we would recommend to the user to do what precisely?

File: docs/source/installation.rst [hunk 2]
```diff
@@ -224,7 +224,15 @@ And build sktime from this environment:
...
+.. warning::
+
+    Some users have experienced issues when installing NumPy, particuarly version 1.19.4. Note that a recent Windows update may affect compilation using Visual Studio (see `Windows update issue <https://developercommunity.visualstudio.com/content/problem/1207405/fmod-after-an-update-to-windows-2004-is-causing-a.html>`_).
+
+.. note::
+
+    If you want to use linux terminal on a windows machine, you can download wsl-2 to experience the same features as linux Ubuntu OS,
+    Make sure you have Ubuntu 20.04 installed on your windows and follow the instructions/requirements to install wsl-2 from `here <https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-2---check-requirements-for-running-wsl-2>`_ and then, run the commands as mentioned above on LINUX section. Maybe you need to install fbprophet dependency at some point using
+    ``pip3 install fbprophet``, if you find build error installing ``pip3 install --editable``.
```
Comment (afzal442) [inline docs/source/installation.rst:235]: @fkiraly I could change something (rephrasing) like this as below

If you find a build error while installing ``pip3 install --editable``, you may need to install fbprophet package at some point using ``pip3 install fbprophet``.

Plz Franz, suggest here. Thanks

**Review by afzal442 — COMMENTED** [review]
Plz, suggest here.

**Comment by afzal442** [comment]
Closing in favour of #1103

**Pull request closed without merging**